components/navigation/DocsNav.js

```diff
@@ -1,7 +1,12 @@
+import React, { useState } from 'react';
 import DocsNavItem from './DocsNavItem';
 import IconHome from '../icons/Home';
+import SubCategoryDocsNav from './SubCategoryDocsNav';
+import DocsArrow from '../icons/DocsArrow';
 
 import { buckets } from '../data/buckets';
+import { useEffect } from 'react';
```

Comment:
I think we should have this in the 2nd line so that we have everything listed in a systematic way  😅